keyboards/planck/keymaps/skug/config.h

```diff
@@ -0,0 +1,49 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first 3 likes (everything up to and including the include) with just `#pragma once`?